src/main/kotlin/kweb/Element.kt

```diff
@@ -76,8 +76,8 @@ open class Element(open val browser: WebBrowser, val creator: ElementCreator<*>?
         if (value != null) {
             val htmlDoc = browser.htmlDocument.get()
             when {
-                htmlDoc != null -> {
-                    val el = htmlDoc.getElementById(this.id!!).attr(name, value.toJson())
+                htmlDoc != null && this.id!= null -> {
```

Comment:
This is an improvement, but will fail silently for elements without an id - the solution may be to require that all elements have an id (although this would be annoying for `<body>` and `<head>`.

Reply:
But doesn't it drop to the else branch below, where it will be executed on the `jsExpression` ? And wasn't that the previous behavior before https://github.com/kwebio/kweb-core/commit/4b42f9eced57d40e6a8989a5bfa3acdd40db14c1 ?

Reply:
Ah, you're right - ok - will merge.